src/Symfony/Component/Config/Definition/Builder/ParentNodeDefinitionInterface.php

```diff
@@ -16,7 +16,7 @@
  *
  * @author Victor Berchet <victor@suumit.com>
  *
- * @method NodeDefinition[] getChildNodeDefinitions() should be implemented since 4.1
+ * @method NodeDefinition[] getChildNodeDefinitions()
```

Comment:
Gets the child node definitions - no implementing it is deprecated since Symfony 4.2